lib/rspec/matchers/expecteds_for_multiple_diffs.rb

```diff
@@ -69,7 +69,12 @@ def diffs(differ, actual)
         @expected_list.map do |(expected, diff_label)|
           diff = differ.diff(actual, expected)
           next if diff.strip.empty?
-          "#{diff_label}#{diff}"
+          if diff == "\e[0m\n\e[0m"
+            "#{DEFAULT_DIFF_LABEL}\n" +
```

Comment:
Why not `diff_label`?

Reply:
True. This is cleaner. Thanks for the review.